src/routes/(authenticated)/app/dashboard/+page.svelte

```diff
@@ -0,0 +1,141 @@
+	import {
+		Border,
+		BorderColor,
+		BorderRounded,
+		ContainerGap,
+		ContainerPadding,
+		Justify,
+	} from '$lib/types'
+	import JoyButton from '$lib/components/Base/Button/JoyButton.svelte'
+	import { ButtonVariant } from '$lib/components/Base/Button'
+</script>
+
+<JoyContainer
+	class="w-full overflow-y-auto"
+	col={true}
+	padding={ContainerPadding.SM}
+	gap={ContainerGap.Y_LG}
+>
+	<JoyContainer class="w-full" padding={ContainerPadding.SM}>
+		<JoyContainer col={true}>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS}>Thursday, August 29</JoyText>
+			<JoyText
+				tag={TextTag.H1}
+				weight={FontWeight.BOLD}
+				size={TextSize.XL_3}
+				color={TextColor.PRIMARY}>Hi John</JoyText
+			>
+		</JoyContainer>
+	</JoyContainer>
+
+	<div class="grid grid-cols-2 grid-flow-row gap-10 w-full">
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Completed Tasks</JoyText>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS} color={TextColor.PRIMARY}
+				>filter</JoyText
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Upcoming Tasks</JoyText>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS} color={TextColor.PRIMARY}
+				>filter</JoyText
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Due Today</JoyText>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS} color={TextColor.PRIMARY}
+				>filter</JoyText
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Overdue Tasks</JoyText>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS} color={TextColor.PRIMARY}
+				>filter</JoyText
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+	</div>
+
+	<JoyContainer
+		class="w-full"
+		col={true}
+		border={Border.XXS}
+		rounded={BorderRounded.XL}
+		padding={ContainerPadding.MD}
+	>
+		<JoyText tag={TextTag.H1} weight={FontWeight.BOLD} size={TextSize.SM}
+			>Recent Project(s)</JoyText
+		>
+		<JoyContainer>
+			<span class="w-12 aspect-square rounded-lg bg-gray-400"> </span>
+			<JoyContainer col={true}>
+				<JoyText tag={TextTag.H1} size={TextSize.XS} weight={FontWeight.BOLD}>
+					Collab Learn
```

Comment:
Hello @arielrecto , can we use localization 'common.platform.name' for this label. Thanks!